d2/src/main/java/com/linkedin/d2/balancer/properties/ClusterStoreProperties.java

```diff
@@ -220,16 +220,16 @@ public boolean equals(Object obj)
       return false;
     }
 
-    ClusterStoreProperties other = (ClusterStoreProperties) obj;
-    if (_clusterFailoutProperties == null && other.getClusterFailoutProperties() != null)
-    {
+    return (obj instanceof ClusterStoreProperties) && canaryEquals((ClusterStoreProperties) obj) && failoutEquals((ClusterStoreProperties) obj);
+  }
+
+  private boolean failoutEquals(ClusterStoreProperties other)
+  {
+    if (_failoutProperties == null && other.getFailoutProperties() != null)
       return false;
-    }
-    if (_clusterFailoutProperties != null && !_clusterFailoutProperties.equals(other.getClusterFailoutProperties()))
-    {
+    if (_failoutProperties != null && !_failoutProperties.equals(other.getFailoutProperties()))
```

Comment:
Our general Java style guide requires braces around control statements for single lines.